metrics/tarantool/info.lua

```diff
@@ -23,9 +23,17 @@ local function update_info_metrics()
 
     for k, v in ipairs(info.replication) do
         if v.upstream ~= nil then
-            local metric_name = 'replication_' .. k .. '_lag'
+            local metric_name_old = 'replication_' .. k .. '_lag'
+            collectors_list[metric_name_old] =
+                utils.set_gauge(metric_name_old, 'Replication lag for instance ' .. k, v.upstream.lag)
+            local metric_name = 'replication_upstream_' .. k .. '_lag'
             collectors_list[metric_name] =
-                utils.set_gauge(metric_name, 'Replication lag for instance ' .. k, v.upstream.lag)
+                utils.set_gauge(metric_name, 'Replication upstream lag for instance ' .. k, v.upstream.lag)
+        end
+        if v.downstream ~= nil then
+            local metric_name = 'replication_downstream' .. k .. '_lag'
```

Comment:
Seems that it should be a lablel. We shouldn't hardcode instance uuid to metric_name.